app/service.js

```diff
@@ -0,0 +1,184 @@
+            },{
+            url: 'https://msdn.microsoft.com/en-us/library/kx37x362.aspx',
+			title: 'C#',
+			img: imgRoot + 'csharp.png',
+            type: 'tool'                                  
+            },{
+            url: 'https://github.com/',
+			title: 'GitHub',
+			img: imgRoot + 'github.png',
+            type: 'tool'                                     
+            },{
+            url: 'https://www.microsoft.com/net/default.aspx',
+			title: 'Microsoft.NET',
+			img: imgRoot + 'dotnet.png',
+            type: 'tool'                                       
+            },{
+            url: 'https://www.visualstudio.com/',
+			title: 'Visual Studio',
+			img: imgRoot + 'vs.png',
+            type: 'tool'                                      
+            },{
+            url: 'https://www.jetbrains.com/idea/',
+			title: 'Intellij IDEA',
+			img: imgRoot + 'idea.png',
+            type: 'tool'                                                                 
+            },{
+            url: "https://www.java.com/",
+			title: "Java",
+			img: imgRoot + 'java.png',
+            type: 'exp'
+            },{
+            url: "http://developer.android.com/index.html",
+			title: "Android Studio",
+			img: imgRoot + 'as.png',
+            type: 'exp'
+            },{
+            url: "https://nodejs.org/",
+			title: "Node.JS",
+			img: imgRoot + 'nodejs.png',
+            type: 'exp'
+          }];
+     
+   
+     this.getTools = function(value){
+         var list = [];
+         for (var i = 0; i < tools.length; i++) {
+             if (tools[i].type == value) {
+                 list.push(tools[i]);
+             };
+         };
+         return list;
+     };
+     
+     
+     skills = [{
+       item: "Creative problem solver",
+       stars: star(5),
+       type: "main"
+       },{       
+       item: "Communication & negotiation",
+       stars: star(5),
+       type: "skill"
+       },{
+        item: "Staff management",
+       stars: star(4),
+       type: "skill"
+       },{
+       item: "Project management",
+       stars: star(4),
+       type: "skill"
+       },{
+       item: "Analitycal skills",
+       stars: star(5),
+       type: "skill"
+       },{
+       item: "Time management",
+       stars: star(4),
+       type: "skill" 
+       },{
+       item: "Russian",
+       stars: star(5),
+       type: "language"
+       },{
+       item: "English",
+       stars: star(4),
+       type: "language"    
+       }];
+     
+     
+     function star(num) {
+         var stars = [];
+         for (var i = 1; i < 6; i++) {
+             var x = "glyphicon glyphicon-star";
+             if(i > num){
+               x = x + ' lite';  
+             };
+             stars.push(x);
+         };
+         return stars;
+     };
+                       
+     this.getSkills = function(value){
+         var list = [];
+         for (var i = 0; i < skills.length; i++) {
+             if (skills[i].type == value) {
```

Comment:
Please, use triple equal (===) to compare values in javascript, read more here http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons